tslint.json

```diff
@@ -27,10 +27,10 @@
       "ignore-bound-class-methods"
     ],
     "switch-default": false,
-    "trailing-comma": [
-      true,
-      { "multiline": "always", "singleline": "never" }
-    ],
-    "no-unused-variable": [true, {"ignore-pattern": "React" }]
+    "no-unused-variable": [true, {"ignore-pattern": "React" }],
+    "trailing-comma": [false],
+    "no-console": [
+      ["error",  ["warn", "error"]]
+    ]
```

Comment:
This is work highlighting - I couldn't get `tslint` and our trailing commas to act in a consistent way (prettier is applied, then `tslint --fix`) and later that fails, in our case this should be fine as prettier will be running on commits and validated at CI